### DISCUSSION TITLE
connect velero with s3 NETAPP

### DISCUSSION THREAD

**Discussion by levwainstein** [vmware-tanzu/velero#4309] [Community support Q&A]
Hi there, 
I'm currently trying to install Velero together with our ONTAP S3 (NETAPP).
https://docs.netapp.com/ontap-9/index.jsp?topic=%2Fcom.netapp.doc.pow-s3-cg%2FGUID-4227B881-18D3-42C1-9941-00D325B074E0.html

is there a chance that anyone did this kind of integration before?

**Comment by zubron** [comment]
Hi @levwainstein - If it is S3 compatible, then you should be able to use the [AWS plugin](https://github.com/vmware-tanzu/velero-plugin-for-aws) which allows for storing backups in S3 compatible storage.
